pyEpiabm/pyEpiabm/intervention/abstract_intervention.py

```diff
@@ -0,0 +1,68 @@
+#
+# AbstractIntervention Class
+#
+
+
+class AbstractIntervention:
+    """Abstract class for Interventions.
+
+    """
+    def __init__(self, start_time, policy_duration, case_threshold,
+                 population, **kwargs):
+        """Set the parameters of the interventions.
```

Comment:
Can we link the interventions wiki in this docstring maybe, or in the docstrings for each intervention? Just so it is easily findable!

Reply:
We put it in all interventions and abstract!